atlasdb-workload-server-api/src/main/java/com/palantir/atlasdb/workload/transaction/witnessed/InvalidWitnessedTransactionAction.java

```diff
@@ -16,18 +16,4 @@
 
 package com.palantir.atlasdb.workload.transaction.witnessed;
 
-import com.palantir.atlasdb.workload.invariant.MismatchedValue;
-import org.immutables.value.Value;
-
-@Value.Immutable
-public interface InvalidWitnessedTransactionAction {
-    @Value.Parameter
-    WitnessedTransactionAction action();
-
-    @Value.Parameter
-    MismatchedValue mismatchedValue();
-
-    static InvalidWitnessedTransactionAction of(WitnessedTransactionAction action, MismatchedValue mismatchedValue) {
-        return ImmutableInvalidWitnessedTransactionAction.of(action, mismatchedValue);
-    }
-}
+public interface InvalidWitnessedTransactionAction {}
```

Comment:
Is this just a marker interface, or do you have plans for it? I checked-out part 7 and it was still empty there.